From: Andrew Morton <akpm@linux-foundation.org>
Date: Wed, 7 Oct 2026 17:06:09 -0700
Subject: [PATCH] 
 lib-cmdline-fix-get_options-count-and-overflow-with-large-ranges-fix

add additional test case from Leizhen

Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Cc: Andy Shevchenko <andriy.shevchenko@intel.com>
Cc: Armaan Sandhu <armaan.sandhu0504@gmail.com>
Cc: Leizhen Zhang <lzsx618@gmail.com>
---
 lib/tests/cmdline_kunit.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/tests/cmdline_kunit.c b/lib/tests/cmdline_kunit.c
index 584fcb2c0e44..12b85f255b65 100644
--- a/lib/tests/cmdline_kunit.c
+++ b/lib/tests/cmdline_kunit.c
@@ -151,6 +151,7 @@ static const struct {
 	{ "0-2147483647",  { 3, 0, 1, 2, },    0,          },
 	{ "-5-2147483647", { 3, -5, -4, -3, }, 0,          },
 	{ "2147483647--5", { 0, 2147483647, }, 0,          },
+	{ "1,0-2147483647", { 3, 1, 0, 1, },   1, },
 };
 
 static void cmdline_test_range_overflow(struct kunit *test)
-- 
2.47.3

